src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -201,6 +232,28 @@ public static void CheckStorageFormat(string format, NetheriteOrchestrationServi
                 {
                     throw new NetheriteConfigurationException($"The current storage format version (={StorageFormatVersion.Last()}) is incompatible with the existing taskhub (={taskhubFormat.FormatVersion}).");
                 }
+
+                // read the immutable log parameters from storage, and keep them as a tuning parameter
+                // (which may override any tuning parameters set by the user; that is what we want since they cannot be honored)
```

Comment:
can we throw a warning if we detect this conflict: that the new parameters cannot be honored?

Reply:
That is a good idea. I think we used to have a warning like that for the partition count (which is also an immutable parameter) but I can't find it anymore in the code. It should be there also.